README.md

```diff
@@ -17,6 +17,10 @@ ___
 The project is comprised of three main components:
 ### <a name="menas"/>Menas
 This is the user-facing web client, used to **specify the standardization schema**, and **define the steps required to conform** a dataset.  
+There are three models used to do this:
+ - **Dataset**: Specifies where the dataset will be read from on HDFS (**RAW**), the conformance rules that will be applied to it, and where it will land on HDFS once it is conformed (**PUBLISH**)
+ - **Schema**: Specifies the schema towards which the dataset will be standardized
+ - **Mapping Table**: Specifies where master reference data can be found (parquet on HDFS) 
```

Comment:
I am not sure what is meant by this line. It would not explain the mapping table to me.
Sorry, I cannot come up with anything better atm.

Reply:
How about now?